WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Package and Rate Selection/WooShippingAddCustomPackageViewModel.swift

```diff
@@ -92,6 +111,9 @@ extension WooShippingPackageDimensionType {
         static let height = NSLocalizedString("wooShipping.createLabel.addPackage.height",
                                               value: "Height",
                                               comment: "Info label for height input field")
+        static let packageWeight = NSLocalizedString("wooShipping.createLabel.addPackage.packageWeight",
+                                              value: "Package Weight",
```

Comment:
nit: We use sentence case (only capitalizing the first letter) for the package type, so I'd do the same thing here instead of capitalizing both words:

```suggestion
                                              value: "Package weight",
```